tracer/src/Datadog.Trace/Debugger/Expressions/ProbeExpressionParser.cs

```diff
@@ -231,12 +231,12 @@ private Expression ConditionalOperator(JsonTextReader reader, Combiner combiner,
                                     }
 
                                 // collection operations
-                                case "hasAny":
+                                case "any":
                                     {
                                         return HasAny(reader, parameters);
                                     }
 
-                                case "hasAll":
+                                case "all":
                                     {
                                         return HasAll(reader, parameters);
                                     }
```

Comment:
Same comment as before - is it a breaking change? Does `hasAny` / `hasAll` exposed to the customer in our docs? If yes, then I would say accept all of them. `hasAny` == `any`, `hasAll` == `all`

Reply:
Fixed in 3bcfaff0516e5c15d9b832a9e3b59d68fcaff851